src/app/api/[[...route]]/routes/rotation.route.ts

```diff
@@ -1,14 +1,35 @@
+import { PrismaClient } from '@prisma/client';
 import { Hono } from 'hono';
 
 const app = new Hono();
+const prisma = new PrismaClient();
 
 export default app;
 
-app.get('/current/:shareHouseId', (c) => {
+app.get('/current/:shareHouseId', async (c) => {
   const shareHouseId = c.req.param('shareHouseId');
-  return c.json({
-    message: `Share house id for current rotation: ${shareHouseId}`,
-  });
+  try {
+    const shareHouseWithAssignmentSheet = await prisma.shareHouse.findUnique({
+      where: {
+        id: shareHouseId,
+      },
+      include: {
+        assignmentSheet: true,
+      },
+    });
+
+    if (
+      !shareHouseWithAssignmentSheet ||
+      !shareHouseWithAssignmentSheet.assignmentSheet
+    ) {
+      return c.json({ error: 'ShareHouse or AssignmentSheet not found' }, 404);
+    }
+
+    return c.json(shareHouseWithAssignmentSheet?.assignmentSheet);
```

Comment:
```suggestion
    return c.json(shareHouseWithAssignmentSheet.assignmentSheet);
```

You can remove the optional changing operator because you return an error if it does not exist.

Reply:
I removed `?` from the sentence.